frontend/src/components/MovieList.tsx

```diff
@@ -30,14 +30,19 @@ const MovieListContainer = styled.div`
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  height: 30em;
+  height: auto;
 `;
 
 const Emoji = styled.span`
   align-self: center;
   font-size: 24px;
 `;
 
+const UL = styled.ul`
```

Comment:
Try to find a better naming. For example: `MovieList` or something. And then the `li` could be named `MovieListItem`. Try to be descriptive.